etc/catalog/memory.properties

```diff
@@ -0,0 +1 @@
+connector.name=memory
```

Comment:
I tried to use the *memory* connector to test multi-statement transactions but it did not look to work. I will remove this.